.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,28 @@
+name: "Linting and Testing"
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, ready_for_review]
```

Comment:
What's the motivation for including `opened` and `ready_for_review`?

Reply:
I think by default it will run on `opened`, `synchronize`, and `reopen`. `synchronize` will be the main one that gets used most of the time but I don't think it triggers until a new commit is pushed to a PR, necessitating `opened` for new PRs. I could see how `reopened` and `ready_for_review` might not be entirely necessary, since I imagine the code shouldn't be changing since the last commit (and if it does need to change, that's when we merge `main` which should trigger `synchronize`). I'm new to configuring Actions, so let me know if I'm off-base at all here.

Reply:
Whoops! Yeah I meant to say "what's the motivation for including `reopened` and ready_for_review`, not `opened`. We can leave them in but I'm pretty sure `reopened` and `ready_for_review` are unnecessary since I'm pretty sure CI will run as normal on draft PRs. I'm also fine leaving them in and removing them if they cause trouble, up to @hinzed1127.

Reply:
Being new to GH actions, I based this off of the [vets-website PR workflow](https://github.com/department-of-veterans-affairs/vets-website/blob/4bd99dd65c152c1cfed545af9b71bed25484bdac/.github/workflows/pull-request.yml#L5). A closer look at the [docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) confirms Louis' thoughts (i.e., it runs on `opened`, `synchronize`, and `reopen` by default).  

I'm fine letting it go off by the defaults and just noting them as a comment in the file. `ready_for_review` would just get triggered after removing draft status from a PR, but it would still have checks run with every push, so the extra trigger seems redundant.